Makefile

```diff
@@ -124,19 +124,22 @@ test-cpp:
 setup-cpp-full-mod: 
 	sudo apt install -y valgrind gdb
 
+setup-cpp-debug: 
+	sudo apt install -y valgrind gdb
+
 # Linux only
 test-cpp-full-mod-asan: build-asan
-	viam-cartographer/build/unit_tests -p -l all -t CartoFacade_io -t CartoFacadeCPPAPI
+	viam-cartographer/build/unit_tests -p -l all
 
 test-cpp-full-mod-valgrind: build-debug
-	valgrind --error-exitcode=1 --leak-check=full -s viam-cartographer/build/unit_tests -p -l all -t CartoFacade_io -t CartoFacadeCPPAPI
+	valgrind --error-exitcode=1 --leak-check=full -s viam-cartographer/build/unit_tests -p -l all
 
 # Linux only
 test-cpp-full-mod-gdb: build-debug
-	gdb --batch --ex run --ex bt --ex q --args viam-cartographer/build/unit_tests -p -l all -t CartoFacadeCPPAPI
+	gdb --batch --ex run --ex bt --ex q --args viam-cartographer/build/unit_tests -p -l all
 
 test-cpp-full-mod: build-debug
-	viam-cartographer/build/unit_tests -p -l all -t CartoFacadeCPPAPI
+	viam-cartographer/build/unit_tests -p -l all
```

Comment:
could you explain what these change do?

Reply:
removing ` -t CartoFacadeCPPAPI` makes it so that all C++ tests will be run, rather than just the test called `CartoFacadeCPPAPI`